plugins/jobs/src/js/JobsRunNow.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable no-unused-vars */
+import React from "react";
+/* eslint-enable no-unused-vars */
+
+import "rxjs/add/operator/switchMap";
```

Comment:
We don't need a switchMap here as far as I can see

Reply:
Yeap. It is reminiscent before I did the the take(1). Thanks.